### PR TITLE
prompt improvement

### PR DESCRIPTION
**Pull request by sonichi**
1. Improve the prompt for the assistant agent and the group chat manager.
2. Bump version to 2.0.1

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/FLAML/. See https://microsoft.github.io/FLAML/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [x] I've made sure all auto checks have passed.

### CODE REVIEW

**Comment by qingyun-wu** [comment]
I guess the update of this prompt requires the update of all the notebook examples.

File: flaml/autogen/agentchat/groupchat.py [hunk 1]
```diff
@@ -41,7 +41,15 @@ def select_speaker_msg(self):
     def select_speaker(self, last_speaker: Agent, selctor: ResponsiveAgent):
```
Comment (LittleLittleCloud) [inline flaml/autogen/agentchat/groupchat.py:41]: selctor -> selector

Comment (sonichi) [reply flaml/autogen/agentchat/groupchat.py]: Thanks. Fixed.

**Review by LittleLittleCloud — APPROVED** [review]
lgtm

**Review by JieyuZ2 — APPROVED** [review]

**Pull request merged**